app.js

```diff
@@ -10,15 +10,100 @@ function CloseBar() {
 }
 
 function showPopUp(a) {
-  const pop = document.getElementById('popUps');
-  pop.style.display = 'flex';
-  if (a === 'tonic') {
-    console.log(a);
-  } else {
-    console.log(a);
+  // popUp container is created with a class assign
+  const popUps = document.createElement('div');
+  popUps.id = 'popUps';
+
+  // popUp content is created with the class assign
+  const popContent = document.createElement('div');
+  popContent.className = 'popContent';
+
+  // poUp heading created
+  const popHeading = document.createElement('div');
+  popHeading.className = 'popHeading';
+
+  const title = document.createElement('h1');
+  const close = document.createElement('h3');
+
+  title.innerText = a;
+  close.innerText = 'x';
+
+  popHeading.appendChild(title);
+  popHeading.appendChild(close);
+
+  close.addEventListener('click', () => {
```

Comment:
- [ ] Please, kindly ensure that your popup data is generated dynamically using javascript as specified by the previous reviewer by creating an array of objects to store the information for each popup data and then using the array containing the object to populate the DOM 🤓 .

Here is an example of the array that will contain the popup informations👇 
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},
{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}
]
```